File: examples/javascript/constructs/for_loop_continue.js
// Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
let i = 0;
while (i < 11) {
    if (i === 5) {
        i = i + 1;
        continue;
    }
    process.stdout.write(String(i));
    if (i < 10) {
        process.stdout.write(", ");
    }
    i = i + 1;
}
console.log("");

File: examples/rust/constructs/for_loop_continue.rs
// Ported from examples/lumen/constructs/for_loop_continue.lm by scripts/port_examples.py; edit the Lumen original, not this file.
fn main() {
    for i in 0..11 {
        if i == 5 {
            continue;
        }
        print!("{}", i);
        if i < 10 {
            print!(", ");
        }
    }
    println!("");
}
